Code completion in completion2.xml for gravity="left|bottom|cen^ selecting center_horizontal:
@@ -4 +4
-     <TextView android:gravity="left|bottom|cen^"></TextView>
+     <TextView android:gravity="left|bottom|center_horizontal^"></TextView>
